lib/libutils/ext/mempool.c

```diff
@@ -0,0 +1,226 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2014, STMicroelectronics International N.V.
+ * All rights reserved.
+ * Copyright (c) 2018, Linaro Limited
+ *
+ * Redistribution and use in source and binary forms, with or without
```

Comment:
Just to clarify: there is SPDX identifier already present. Do you need a full license text?

Reply:
In this case the full license text is needed since I can't speak for STMicro and they are copyright holders as well.